.github/workflows/examples.yml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

The current `examples` CI is executed only once a day. This is because it takes several hours to execute all examples sequentially. So, it is difficult for us to find and fix the errors using it. How about just focusing on the integration modules in this PR?

I'm trying to improve the example CI now in #2534, which splits the workflow into different workflows. After that, we can test individual examples, and it'll be easier to introduce Python 3.9 to the `examples` CI.

CC @nzw0301 

```suggestion
        python-version: [3.6, 3.7, 3.8]
```